system: clean up windows dashboards

efd6 opened the pull request:
<!-- Type of change
Please label this PR with one of the following labels, depending on the scope of your change:
- Bug
- Enhancement
- Breaking change
- Deprecation
-->

## What does this PR do?

<!-- Mandatory
Explain here the changes you made on the PR.
-->

This does a general clean up of the Windows dashboards. Each commit is a distinct operation.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

<!-- Recommended
Add a checklist of things that are required to be reviewed in order to have the PR approved
-->
- [ ]

## How to test this PR locally

<!-- Recommended
Explain here how this PR will be tested by the reviewer: commands, dependencies, steps, etc.
-->

## Related issues

<!-- Recommended
Link related issues below. Insert the issue link or reference after the word "Closes" if merging this should automatically close it.

- Closes #123
- Relates #123
- Requires #123
- Supersedes #123
-->
- For #5022

## Screenshots

![system-windows-overview](https://user-images.githubusercontent.com/4565752/227271531-0bbb5c1e-f867-4793-afa5-248f7d560023.png)
![system-windows-security-user-logons](https://user-images.githubusercontent.com/4565752/227271548-5e8292d3-f63b-47dc-8fe1-fec4b92eb3ed.png)
![system-windows-security-failed-and-blocked-accounts](https://user-images.githubusercontent.com/4565752/227271581-142ae893-ac56-4f1c-9d2e-a80770748ade.png)
![system-windows-security-user-management-events](https://user-images.githubusercontent.com/4565752/227271641-691171ad-71e5-4483-b4a0-59190335ff2e.png)
![system-windows-security-group-management-events](https://user-images.githubusercontent.com/4565752/227271661-14d33834-3ce4-4eb9-a966-603cd3daa85e.png)

elasticmachine commented:
## :robot: GitHub comments

<details><summary>Expand to view the GitHub comments</summary>
<p>

To re-run your PR in the CI, just comment with:



- `/test` : Re-trigger the build.




</p>
</details>

<!--COMMENT_GENERATED_WITH_ID_comment.id-->

elasticmachine commented:
### :globe_with_meridians: Coverage report

Name | Metrics % (`covered/total`) | Diff
 --- | --- | ---
Packages | 100.0% (`3/3`) | :green_heart: 
Files | 100.0% (`4/4`) | :green_heart: 3.276
Classes | 100.0% (`4/4`) | :green_heart: 3.276
Methods | 60.759% (`48/79`) | :-1: -31.028
Lines | 99.535% (`2780/2793`) | :+1: 8.24
Conditionals | 100.0% (`0/0`) | :green_heart: 

<!--COMMENT_GENERATED_WITH_ID_coverage-->

elasticmachine commented:
Pinging @elastic/security-external-integrations (Team:Security-External Integrations)

andrewkroh reviewed (approved):
If modifying the dashboard filters is in scope for this PR, then for `[System] Windows Overview` can you add `winlog.winlog` to include the default data_stream.dataset used by the Custom Windows events input package.

The other four `[System Windows Security] *` dashboards are all looking at data from `winlog.provider_name: Microsoft-Windows-Security-Auditing`. So I think we should add that as a dashboard filter. I think those four should also include `windows.forwarded` in the data_stream.dataset filter since we can have forwarded auditing events in that channel.

I posted some screenshots to the description. I used the `elastic-package install` command to install this to https://kibana.siem.estc.dev.

efd6 commented:
@andrewkroh I've added those (caution: added manually since I don't have an easy windows datastream here, so winlog.* gives a warning on the dashboard). PTAL
